styles/components/nav.css

```diff
@@ -10,4 +10,10 @@
   border-top-right-radius: 1em;
   margin-top: 2em;
   padding: 1.5rem;
+  z-index: 2000;
```

Comment:
haha this is a quite high number 😄

Reply:
haha yeah the bookmark was in the front of the navbar, so i put this on 2000, that the navbar is ALWAYS in the front of everything... 😂